{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2018-7886",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "An issue was discovered in CloudMe 1.11.0. An unauthenticated local attacker that can connect to the \"CloudMe Sync\" client application listening on 127.0.0.1 port 8888 can send a malicious payload causing a buffer overflow condition. This will result in code execution, as demonstrated by a TCP reverse shell, or a crash. NOTE: this vulnerability exists because of an incomplete fix for CVE-2018-6892."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "44470",
                "refsource": "EXPLOIT-DB",
                "url": "https://www.exploit-db.com/exploits/44470/"
            },
            {
                "name": "https://0day4u.wordpress.com/2018/03/09/buffer-overflow-on-cloudme-sync-v1-11-0/",
                "refsource": "MISC",
                "url": "https://0day4u.wordpress.com/2018/03/09/buffer-overflow-on-cloudme-sync-v1-11-0/"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}